paf-mvp-frontend/rollup.config.js

```diff
@@ -130,7 +130,7 @@ export default [
             serve({ // dev server
               contentBase: '',
               headers: {
-                'Access-Control-Allow-Origin': '*'
+                'Access-Control-Allow-Origin': '*',
```

Comment:
NIT apparently you planned to change this file but in the end, you didn't need to. Maybe cleaner to rollback this change.